webapp/src/ts/selectors/index.ts

```diff
@@ -51,6 +52,12 @@ export const Selectors = {
   getMessagesError: createSelector(getMessagesState, (messagesState) => messagesState.error),
   getSelectedConversation: createSelector(getMessagesState, (messagesState) => messagesState.selected),
   getConversations: createSelector(getMessagesState, (messagesState) => messagesState.conversations),
+
+  // contacts
+  getContactsList: createSelector(getContactsState, (contactsState) => contactsState.contacts.contacts),
+  contactListContains: createSelector(getContactsState, (contactsState) => {
+    return (id) => contactsState.reportsById.has(id);
```

Comment:
Umm, I don't think `contactsState.reportsById` exists. Looks like you copied this from reports without implementing the actual properties in the state. 
Also, shouldn't this be `contactsById`?